content/en/docs/languages/java/api-components.md

```diff
@@ -0,0 +1,1690 @@
+                Arrays.asList("value1", "value2"))
+            .setAttribute(
+                AttributeKey.booleanArrayKey("come.acme.bool-array-key"),
+                Arrays.asList(true, false))
+            .setAttribute(
+                AttributeKey.longArrayKey("come.acme.long-array-key"), Arrays.asList(1L, 2L))
+            .setAttribute(
+                AttributeKey.doubleArrayKey("come.acme.double-array-key"), Arrays.asList(1.1, 2.2))
+            // Optionally omit initializing AttributeKey
+            .setAttribute("com.acme.string-key", "value")
+            .setAttribute("com.acme.bool-key", true)
+            .setAttribute("come.acme.long-key", 1L)
+            .setAttribute("come.acme.double-key", 1.1)
+            .setAllAttributes(WIDGET_RED_CIRCLE)
+            // Uncomment to optionally explicitly set the parent span context. If omitted, the
+            // span's parent will be set using Context.current().
+            // .setParent(parentContext)
+            // Uncomment to optionally add links.
+            // .addLink(linkContext, linkAttributes)
+            // Start the span
+            .startSpan();
+
+    // Check if span is recording before computing additional data
+    if (span.isRecording()) {
+      // Update the span name with information not available when starting
+      span.updateName("new span name");
+
+      // Add additional attributes not available when starting
+      span.setAttribute("com.acme.string-key2", "value");
+
+      // Add additional span links not available when starting
+      span.addLink(exampleLinkContext());
+      // optionally include attributes on the link
+      span.addLink(exampleLinkContext(), WIDGET_RED_CIRCLE);
+
+      // Add span events
+      span.addEvent("my-event");
+      // optionally include attributes on the event
+      span.addEvent("my-event", WIDGET_RED_CIRCLE);
+
+      // Record exception, syntactic sugar for a span event with a specific shape
+      span.recordException(new RuntimeException("error"));
+
+      // Set the span status
+      span.setStatus(StatusCode.OK, "status description");
+    }
+
+    // Finally, end the span
+    span.end();
+  }
+
+  /** Return a dummy link context. */
+  private static SpanContext exampleLinkContext() {
+    return Span.fromContext(current()).getSpanContext();
+  }
+}
+```
+<!-- prettier-ignore-end -->
+
+Span parenting is an important aspect of tracing. Each span has an optional
+parent. By collecting all the spans in a trace and following each span's parent,
+we can construct a hierarchy. The span APIs are build on top of
+[context](#context), which allows span context to be implicitly passed around an
+application and across threads. When a span is created, its parent is set to the
+whatever span is present in `Context.current()` unless there is no span or the
+context is explicitly overridden.
+
+Most of the context API usage guidance applies to span. Span context is
```

Comment:
if this refers to spans in general: 
```suggestion
Most of the context API usage guidance applies to spans. Span context is
```

or if it's referring to the class, maybe we could make that more explicit:
```suggestion
Most of the context API usage guidance applies to `Span`. Span context is
```